.github/CODEOWNERS

```diff
@@ -28,3 +28,6 @@
 
 /bitrise.yml @mozilla-mobile/releng @isabelrios @st3fan
 
+# Changes to string files can be reviewed and approved by mergify
+
+/Blockzilla/**/*.strings @mergify
```

Comment:
If I'm not wrong this patter will cover:

- Blockzilla/*.lproj/*.strings
- Blockzilla/Settings.bundle/*.lproj/*.strings

Do you think this will work @st3fan ?

Reply:
Let's give it a try!

Reply:
It worked! https://github.com/mozilla-mobile/focus-ios/pull/2856 approved and landed automatically \o/